### DISCUSSION TITLE
Unifi G4 Doorbell Pro PoE not working as doorbell accessory in HomeKit

### DISCUSSION THREAD

**Discussion by superpower10** [koush/scrypted#1178] [Q&A]
Can't add the HomeKit-Plugin to the device.

I have seen that there was an [issue](https://github.com/koush/scrypted/issues/959) already that has been moved to a [discussion](https://github.com/koush/scrypted/discussions/960) which has been marked as solved. 

But I couldn't find how or what to do. 

Could somebody please help me?

I'm using scrypted for Home Assistant in Version 0.57.0 and HomeKit-Plugin Version 1.2.30 (which are the newest ones I could find).

<img width="588" alt="Bildschirmfoto 2023-11-07 um 11 54 57" src="https://github.com/koush/scrypted/assets/20533966/34783ea4-d8c6-4bfd-8d54-c3c41380d50e">
<img width="1350" alt="Bildschirmfoto 2023-11-07 um 11 55 22" src="https://github.com/koush/scrypted/assets/20533966/1014f78d-b342-4471-936e-38f1f4fb3c64">

**Comment by frostywite** [comment]
Also having an issue with G4 Pro PoE doorbell. I can add the camera to HomeKit, HomeKit displays the stream on the desktop and can pair the camera. HomeKit is displaying the camera feed but when trying to access the camera it becomes unavailable. Also no option for doorbell activation automation. Originally had a standard G4 doorbell that just died that was working.
